en/installconfig-network-subnet-management.md

```diff
@@ -0,0 +1,50 @@
+Title: Subnet Management | MAAS
+
+
+# Subnet Management 
+
+The topic of subnet management pertains to whether MAAS is in full control of a
```

Comment:
I can't think of anything else. Suggestions? Maybe I should scrub the introduction as there is really not much to say.